canonical/cloud-init
# testing: OCI availability domain is now required (SC-59)

## Proposed Commit Message


```
testing: OCI availability domain is now required
```

## Additional Context
See https://github.com/canonical/pycloudlib/issues/138 and https://github.com/canonical/pycloudlib/pull/147

## Test Steps
Specify 'oci' as your integration testing `PLATFORM`. If you don't set `ORACLE_AVAILABILITY_DOMAIN`, the test exit with an exception that it is required. When `ORACLE_AVAILABILITY_DOMAIN` is specified, the tests should run as expected.

## Checklist:
